__tests__/app/auth/signin/page.test.tsx

```
@@ -0,0 +1,232 @@
+    jest.clearAllMocks();
+    
+    mockUseRouter.mockReturnValue({
+      push: mockPush,
+      replace: jest.fn(),
+      prefetch: jest.fn(),
+      back: jest.fn(),
+      forward: jest.fn(),
+      refresh: jest.fn(),
+    });
+
+    mockUseSearchParams.mockReturnValue(mockSearchParams as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)